### PR TITLE
18. As a user, I want the navbar to be on the top of the page #34

### LINKED ISSUE #34: 18. As a user, I want the navbar to be on the top of the page
### Acceptance Criteria 
- [x] Make the Navbar on the top of the page
- [x] Make the title of the page professional
- [x] Make the user indication on the right side of the navbar -> add icons for this as well
- [x] Highlight the current page you are on

### PR DESCRIPTION
**Pull request by khadijahnabil**
## Description

Changed the look of the navbar

## Related Issue

Closes #34 

## Acceptance Criteria

- Make the Navbar on the top of the page
- Make the title of the page professional
- Make the user indication on the right side of the navbar -> add icons for this as well
- Highlight the current page you are on

## Type of Changes
UI Enhancement

### Before
Screenshot 2024-03-28 at 18 01 10

### After
Screenshot 2024-03-28 at 18 00 36

## Testing Steps / QA Criteria
1. pull the changes from the branch kn-navbar
2. click the sign-in button -> you should see the UI differences
3. click the sign-out button -> you should see the UI differences
4. navigate to different pages by clicking in the navbar -> you should see the page you are on being highlighted

### CODE REVIEW

**Review by ijayhub — CHANGES_REQUESTED** [review]
This is a nice job, but I suggest setting the highlight color of the current page to #e69500. That way, it matches the highlight color I've already chosen for the list.
I like the favicon, nice.

File: src/views/Layout.jsx [hunk 1]
```diff
@@ -20,21 +21,27 @@ export function Layout() {
...
+									<FaUserMinus />
+									<SignOutButton />
+								</span>
+							</div>
+						)}
+					</div>
+					<div className="Layout-title">
+						<FaShoppingBag />
+						<h1>Smart Shopping List</h1>
+					</div>
```
Comment (ijayhub) [inline src/views/Layout.jsx:43]: I think a "smart shopping list App." - title

**Review by jmahamed — APPROVED** [review]
looks great!

**Review by redapy — APPROVED** [review]
LGTM, great work @khadijahnabil !🚀

**Pull request merged**